batch/batch/driver/main.py

```diff
@@ -440,43 +442,144 @@ def validate_int(name, value, predicate, description):
     return web.HTTPFound(deploy_config.external_url('batch-driver', pool_url_path))
 
 
-@routes.get('/inst_coll/{inst_coll}')
+@routes.post('/config-update/job-private/{job_private}')
```

Comment:
Why there are multiple job private instance collections? I was imagining exactly one job private instance collection which managed all the job private instances.